dev-kubeflow-org/ks-app/app.yaml

```diff
@@ -3,32 +3,32 @@ environments:
   default:
     destination:
       namespace: kubeflow
-      server: https://35.188.73.10
+      server: https://10.19.47.136:6443
```

Comment:
This looks like your minikube cluster. We probably need to point at the dev cluster before running recreate_app.sh